spec/controllers/receipts_controller_spec.rb

```diff
@@ -0,0 +1,67 @@
+require "rails_helper"
+
+RSpec.describe ReceiptsController, type: :controller do
+  let!(:user) { FactoryBot.create :user }
+  let!(:receipt) { FactoryBot.create :receipt, user_id: user.id }
+  let!(:receipts) { user.receipts }
+
+  before do
+    log_in user
+  end
+
+  describe "GET receipts#index" do
+    context "success when valid attributes" do
+      before do
+        get :index
+      end
+
+      it "assigns @receipts" do
+        expect(assigns(:receipts)).to eq receipts
+      end
+
+      it {should render_template(:index)}
```

Comment:
```suggestion
      it {expect(response).to render_template :index}
```
dùng expect luôn cho nó thống nhát bé